pkg/apis/stork/v1alpha1/backuplocation.go

```diff
@@ -43,6 +43,7 @@ type BackupLocationItem struct {
 	SecretConfig       string        `json:"secretConfig"`
 	Sync               bool          `json:"sync"`
 	RepositoryPassword string        `json:"repositoryPassword"`
+	EncryptionV2Key    string        `json:"encryptionV2Key"`
```

Comment:
what's different between `EncryptionKey` and `EncryptionV2Key` ?

Reply:
EncryptionKey is deprecated now and we will use EncryptionV2Key for storing the encryption key. Added the comments in the definition.